odpf/meta/facets/Columns.proto

```diff
@@ -0,0 +1,19 @@
+syntax = "proto3";
+
+package odpf.meta.facets;
+
+option java_package = "io.odpf.proton.meta.facets";
+option java_outer_classname = "Columns";
+option go_package = "github.com/odpf/proton/meta/facets";
+
+message Columns {
+  repeated Column columns = 1;
+}
+
+message Column {
+  string name = 1;
+  string description = 2;
+  string data_type = 3;
```

Comment:
why not just `type`?

Reply:
`Column.type` might mean something else while `Column.data_type` clearly talking about the data type as in`varchar` or `int`.
wdyt?

Reply:
Sounds good 👍  but then we should follow this in `length` field as well right?

Reply:
imo column length is pretty common but maybe it's just me and if you didnt catch it the first time probably it needs to be renamed.

what would be clearer, `field_length`, `data_length`?